src/huggingface_hub/keras_mixin.py

```diff
@@ -59,9 +173,13 @@ def save_pretrained_keras(
         with open(path, "w") as f:
             json.dump(config, f)
 
+    hyperparameters = extract_hyperparameters_from_keras(model)
+    plot_network(model, save_directory)
```

Comment:
Maybe let's have something similar to `is_tf_available` but for pydot. We can also add it as a test dependency.